kube/app/ingress-external.yml

```diff
@@ -15,7 +15,7 @@ spec:
         - {{ .DRONE_BUILD_NUMBER }}.{{ .BRANCH_ENV }}.homeoffice.gov.uk
         - {{ .APP_NAME }}-{{ .DRONE_SOURCE_BRANCH }}.{{ .BRANCH_ENV }}.homeoffice.gov.uk
       {{ else if eq .KUBE_NAMESPACE .UAT_ENV }}
-        - dev.notprod.{{ .APP_NAME }}.homeoffice.gov.uk
+        - rotm.uat.internal.sas-notprod.homeoffice.gov.uk
```

Comment:
Any reason you are adding an internal ingress name not an external ingress file @James-Hovell @SarahJaneLuff 😬. Ain't gonna work. You might want to revert this and update the ingress-internal file and deploy that to UAT using drone instead. Might make your life easier :D